### ISSUE TITLE
Editing the `items` property in the px-steps demo causes a stack overflow

### ISSUE THREAD

**Issue by davidrleonard** [predix-ui/predix-ui.github.io#314]
Steps to reproduce: 

1. Switch to polymer2.0-preview branch and run bower install
2. Open px-steps demo (http://localhost:8080/#/elements/px-steps)
3. Click the edit button for the `items` prop
4. Make some small change in the title of one step
5. Tap save

The px-steps component will disappear and the following error will be logged to the console:

```
Uncaught RangeError: Maximum call stack size exceeded
    at marshalArgs (property-effects.html:993)
    at Object.runMethodEffect [as fn] (property-effects.html:813)
    at runEffectsForProperty (property-effects.html:160)
    at runEffects (property-effects.html:126)
    at HTMLElement._propertiesChanged (property-effects.html:1674)
    at HTMLElement._flushProperties (property-accessors.html:564)
    at HTMLElement.__enableOrFlushClients (property-effects.html:1573)
    at HTMLElement._flushClients (property-effects.html:1548)
    at HTMLElement._propertiesChanged (property-effects.html:1670)
    at HTMLElement._flushProperties (property-accessors.html:564)
```

Browser: Chrome 64.0.3282.39
OS: OSX Sierra 10.12
Branch: polymer2.0-preview
Commit tested: f1dde34

![image](https://user-images.githubusercontent.com/6456757/34240084-853d314a-e5c0-11e7-8e9a-4a2f73a4f506.png)

**Comment by davidrleonard** [comment]
I'm working on this.

**Comment by davidrleonard** [comment]
Fixed in https://github.com/PredixDev/px-steps/commit/59fdab5b39b5f47368b34b96376e3c2560ce90e4, released as of px-steps v1.1.1
